samples/bluetooth/peripheral_nucleo_wb55rg/prj.conf

```diff
@@ -0,0 +1,13 @@
+CONFIG_BT=y
+CONFIG_BT_DEBUG_LOG=y
+CONFIG_BT_PERIPHERAL=y
+CONFIG_BT_DEVICE_NAME="P2PSRV1"
+CONFIG_BT_GATT_BAS=y
+CONFIG_BT_GATT_CLIENT=y
+CONFIG_BT_DEBUG=y
+CONFIG_GPIO=y
```

Comment:
GPIO should not be needed. This is part of the few drivers part of default board configuration

Reply:
done

Reply:
ok